install/ontology/category.rdf

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<rdf:RDF
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+    xml:base="http://www.tao.lu/Ontologies/lom.rdf#">
```

Comment:
@siwane 

I know it's painful, and sorry to make you the first guy in OAT to do that but... it's definitely time to change our xml:base attributes with something more aligned with the current OAT situation. Could you please change "http://www.tao.lu/Ontologies/lom.rdf#" to "http://www.taotesting.com/ontologies/lom.rdf#"?

Thanks a million.